### DISCUSSION TITLE
How I can disable Right click on open-shell-menu

### DISCUSSION THREAD

**Discussion by lechardonneret** [Open-Shell/Open-Shell-Menu#1149] [Q&A] [answered]
**_Hi,
i want to disable right clik  in open shell start menu ,like old classic shell start menu when we create previously a key on registry named "EnableSettings" and we set it to 0 at HKEY_LOCAL_MACHINE\SOFTWARE\IvoSoft\ClassicStartMenu. this is for simple resaon to hide settings that users won't have access .
Thanks for all._**

**Comment by ge0rdi** [comment, marked as answer]
`Open-Shell` settings are pretty much the same as `Classic-Shell` ones. Only the registry path is different.

You should use `HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell\StartMenu`.

**Reply by lechardonneret** [reply]
I tested but it didn't work, still gives access to the right click, also on HKEY_CURRENT_USER\SOFTWARE\OpenShell\StartMenu\Settings ,i added the two key "EnableContextMenu" and "EnableSettings" and i set its two to 0, but no results.

**Reply by ge0rdi** [reply]
It seems to be working for me.

I've created the registry value using `reg add HKLM\SOFTWARE\OpenShell\StartMenu /v EnableSettings /t REG_DWORD /d 0`.
Then installed `Open-Shell` 4.4.170.
And I can't start the settings dialog, nor it is present in right-click context menu.
![image](https://user-images.githubusercontent.com/2453094/191042973-3c118815-62f0-4bcb-b2c5-11a650708e92.png)

What `Open-Shell` version do you have?
Can you test it on clean machine?

**Reply by lechardonneret** [reply]
i have the last version 4.4.178 installed on win10 21H2 ,I will test the 4.4.170 .

**Reply by lechardonneret** [reply]
now , it works for version 4.4.170

**Reply by lechardonneret** [reply]
After uninstall and install the 4.4.178 version ,it work perfectely thank you ge0rdi.
